pkg/admin/facade/user_export.go

```diff
@@ -0,0 +1,19 @@
+package facade
+
+import (
+	"context"
+
+	"github.com/authgear/authgear-server/pkg/lib/userexport"
+)
+
+type UserExportService interface {
+	ExportRecords(ctx context.Context, request *userexport.Request) *userexport.Response
+}
+
+type UserExportFacade struct {
+	Service UserExportService
+}
+
+func (s *UserExportFacade) ExportRecords(ctx context.Context, request *userexport.Request) *userexport.Response {
+	return s.Service.ExportRecords(ctx, request)
+}
```

Comment:
Remove this facade. For now, you can just inject UserExportService in the handler.

Reply:
removed the facade to use worker task similar as user import now